ultra/baselines/ppo/ppo/policy.py

```diff
@@ -0,0 +1,397 @@
+# some parts of this implementation is inspired by https://github.com/Khrylx/PyTorch-RL
```

Comment:
Are the MIT license requirements fulfilled?

Reply:
@Gamenot Could you ask Nhat? he was dealing with licenses back in November